beacon-chain/sync/service.go

```diff
@@ -47,14 +50,35 @@ type Config struct {
 	CrystallizedStateBufferSize     int
 }
 
+// FinalizedBlock stores information regarding a block, its finalized epoch and the contents
+// of its crystallized state.
+type FinalizedBlock struct {
+	BeaconBlock        *types.Block
+	LastFinalizedEpoch uint64
+	CrystallizedState  *types.CrystallizedState
+}
+
 // DefaultConfig provides the default configuration for a sync service.
 func DefaultConfig() Config {
 	return Config{100, 100, 100, 100, 100, 100}
```

Comment:
Can we use the struct key identifiers here instead of short form?

Reply:
alright will make it explicit